packages/peregrine/lib/store/actions/cart/asyncActions.js

```diff
@@ -317,9 +317,9 @@ export const removeItemFromCart = payload => {
 };
 
 export const getCartDetails = payload => {
-    const { apolloClient, fetchCartId, fetchCartDetails } = payload;
+    const { fetchCartId, fetchCartDetails } = payload;
 
-    return async function thunk(dispatch, getState) {
+    return async function thunk(dispatch, getState, { apolloClient }) {
```

Comment:
Now that our thunk middleware includes an `extraArgument` object with the Apollo client, we can access it from any thunk.